ACMILabs/spotlights
# #43 Update for new Labels API, security update

Resolves #43

Setup for local development.

### Acceptance Criteria
- [x] Make sure this project is setup for local development
- [x] Update/add testing and development instructions to README
- [x] Update `package.json` for security updates

### Relevant design files
* None

### Testing instructions
Follow the README to see if you can:
1. Run a development server (Use `XOS_PLAYLIST_ID=19`)
1. Run Python linting and tests
1. Run Javascript linting and tests

### DoD
For requester to complete:
